### PR TITLE
Adjusts styptic powder splashing

### PR DESCRIPTION
**Pull request by gurking**
## About The Pull Request

Removes the healing effect from splashing yourself with styptic powder.

## Why It's Good For The Game

Full brute healing with one click is dumb.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
balance: styptic powder effective reagent volume
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -350,11 +350,18 @@
 			if(show_message)
 				to_chat(M, "<span class='warning'>You don't feel so good...</span>")
 		else if(M.getBruteLoss())
-			M.adjustBruteLoss(-reac_volume)
-			if(show_message)
-				to_chat(M, "<span class='danger'>You feel your bruises healing! It stings like hell!</span>")
-			M.emote("scream")
-			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
+			if (reac_volume >= 21)
```
Comment (lolman360) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:353]: have this check for the TOUCH method, patches shouldn't proc this (they have a fairly large delay and don't go through suits)

Comment (gurking) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:353]: Aren't patches TOUCH method too?

Comment (lolman360) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:353]: patches are SPRAY, funnily enough

**Comment by Kneesox44** [comment]
Wait, doesn't this apply to sprays and patches, meaning that now, use of styptic-powder sprays will just cause tox damage?

**Comment by gurking** [comment]
> Wait, doesn't this apply to sprays and patches, meaning that now, use of styptic-powder sprays will just cause tox damage?

No, this only applies to splashing directly with a beaker etc. Also doesn't deal any tox damage, as it provides no benefit either.

**Pull request merged**